src/modules/roc_packet/target_posix/roc_packet/address.h

```diff
@@ -56,9 +62,15 @@ class Address {
     //! Check whether this is multicast address.
     bool multicast() const;
 
+    //! Check whether multicast interface address was initialized.
+    bool has_multicast_iface() const;
+
     //! Get IP address.
     bool get_ip(char* buf, size_t bufsz) const;
```

Comment:
I suggest to rename `get_ip()`, `set_ip...()` etc to `get_host()`, `set_host...()` etc, to avoid confusion (because multicast interface is also an IP).

Reply:
I'd also rename `multicast_iface` to `miface` for brevity (but not in the public API).